### ISSUE TITLE
Tracks not in order...

### ISSUE THREAD

**Issue by thisisvilla** [Jedidiah/gatsby-source-soundcloud#4]
When getting tracks in a playlist using `allSoundcloudplaylist`, the tracks are returned in the order uploaded, and not in the order dictated by the playlist. Is there anything im missing?

Thanks! :)

**Comment by Jedidiah** [comment]
You're right! I didn't notice as the tracks I was using where uploaded in the same order as they should be in the playlist.

**Comment by thisisvilla** [comment]
haha cool :) im currently hitting the api in the client solely to get the right order of tracks (https://www.ghostparkmusic.com) - it would be nice to not have to do that work-around and have the order returned proper from this plugin. Might that be a possibility? Thanks man!! Great work here!

**Comment by Jedidiah** [comment]
Yeah, will give you a ping when it's in. Or feel free to send a PR. I'm going to be working on it this weekend.

**Comment by Jedidiah** [comment]
Nice site btw 👍

**Comment by thisisvilla** [comment]
Thanks!! :)
